networking/init-cms-bundle/4.0/config/packages/networking_init_cms.yaml

```diff
@@ -0,0 +1,101 @@
+imports:
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/doctrine.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/fos_ck_editor.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/fos_user.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/mopa_bootstrap.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/oneup_flysystem.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/oneup_uploader.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_admin.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_block.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_core.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_doctrine_admin.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_formatter.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_media.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_notification.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_user.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/stof_doctrine_extensions.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/symfony_cmf_routing_extra.yaml" }
+# Default security configuration
+#    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/security.yaml" }
+
+# Set templating to twig. FOSUserBundle requires templating to be setup
+framework:
```

Comment:
Please done change other bundle's configuration

Reply:
This configuration is needed in order fo the CMS to work. The idea is for us to add the configuration so that the user does not need to configure all the individual bundles themselves. 

It is vital that it is done this way.